### ISSUE TITLE
No mention in docs about using `default=colander.drop`

### ISSUE THREAD

**Issue by tisdall** [Pylons/colander#237]
You can apparently use a `colander.drop` as a value for `default`, but there's no mention in the documents.

Example:
```
>>> x = colander.SchemaNode(
...     colander.Mapping(),
...     colander.SchemaNode(colander.String(), name="y")
... )
>>> x.serialize({'y':3})
{'y': '3'}
>>> x.serialize({})
{'y': <colander.null>}
>>> x = colander.SchemaNode(
...     colander.Mapping(),
...     colander.SchemaNode(colander.String(), name="y", default=colander.drop)
... )
>>> x.serialize({})
{}
```

Now, I find this feature very useful for what I'm working on.  However, before I work on updating the docs to mention how this works, does anyone have any reasons for this to be labelled a bug?

**Comment by tisdall** [comment]
Okay, it seems that this is mentioned in the CHANGES.rst but no one has updated the documentation.

**Comment by tonthon** [comment]
colander.drop should be used as value for missing, not default
http://docs.pylonsproject.org/projects/colander/en/latest/api.html#colander.SchemaNode
```
missing: The default deserialization value for this node. If it is not provided, the missing value of this node will be the special marker value colander.required, indicating that it is considered ‘required’. When missing is colander.required, the required computed attribute will be True. When missing is colander.drop, the node is dropped from the schema if it isn’t set during serialization/deserialization.
```

**Comment by tisdall** [comment]
@tonthon - https://github.com/Pylons/colander/blob/master/CHANGES.rst#features-1  point 2.

**Comment by tonthon** [comment]
https://github.com/tisdall - Oh I missed that one, I didn't know colander.drop could be used that way.

**Comment by tisdall** [comment]
@tonthon, and do you know why you didn't know?  Because it's not in the docs!

**Comment by stevepiercy** [comment]
Is there a PR that closes this issue? I can merge a PR for the docs only, but not code as I don't have the necessary familiarity.

**Comment by tisdall** [comment]
@stevepiercy  #238 is sort-of docs only...  ^_^  But, no, I haven't made anything to update the docs regarding this.  I'm not working with colander right now so I likely won't be submitting a PR either so it's open for someone to contribute.

**Comment by mmerickel** [comment]
A PR improving the docs for this feature is likely to be accepted. Any movement on this?

**Comment by lmctv** [comment]
Forgot to put a reference to this issue in the pull request #248

**Comment by tisdall** [comment]
fixed in #248
